pkg/remote/types.go

```diff
@@ -29,3 +31,26 @@ func (n *nopCloser) Write(ctx context.Context, wr *prompb.WriteRequest) error {
 }
 
 func (n *nopCloser) CloseIdleConnections() {}
+
+func WriteRequest(ctx context.Context, clients []RemoteWriteClient, wr *prompb.WriteRequest) error {
+	var wg sync.WaitGroup
+	wg.Add(len(clients))
+	errs := make(chan error, len(clients))
+	for _, client := range clients {
+		go func(client RemoteWriteClient) {
```

Comment:
```suggestion
		client := client
		go func(client RemoteWriteClient) {
```
This might not be necessary depending on the Go version, but probably not a bad idea to add these closure captures until we're well beyond the current critical version.